use/2015-12-15-microsatellite-differentiation.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "Calculating genetic differentiation from SSR data"
+---
+
+# Introduction
+
+This workflow will show you how to assess population differentiation with
+microsatellite/SSR data. We will analyse F-statistics, AMOVA, and Discriminant
```

Comment:
Convert to one sentence: "We will analyze a data set of bovine  samples from France and West Africa using F-statistics, AMOVA, and Discriminant Analysis of Principal Components (DAPC)."